src/Corporation/Actions.ts

```diff
@@ -628,3 +631,37 @@ export function SetProductMarketTA1(product: Product, on: boolean): void {
 export function SetProductMarketTA2(product: Product, on: boolean): void {
   product.marketTa2 = on;
 }
+
+export function Bribe(
+  corporation: Corporation,
+  fundsForBribing: number,
+  factionName: FactionName,
+): {
+  success: boolean;
```

Comment:
Does this need to return an object? Is it not sufficient to assume that 0 rep-gain = failure?